test/run-tests.proj

```diff
@@ -10,7 +10,7 @@
     <Message Importance="High" Text="Running tests ..." />
     <MSBuild Targets="VSTest"
              Projects="@(Project)"
-             Properties="VSTestNoBuild=true" />
+             Properties="VSTestNoBuild=true;Coverage=true;CopyLocalLockFileAssemblies=true" />
```

Comment:
> CopyLocalLockFileAssemblies=true

Is this not a duplicate of the value in **Directory.Build.targets** set whenever `Coverage` is true?